src/ui/hooks/useSeekable.ts

```diff
@@ -21,6 +23,6 @@ export const useSeekable = () => {
     return () => {
       player?.removeEventListener(PlayerEventType.PROGRESS, onUpdateSeekable);
     };
-  }, [player]);
+  }, [player, setSeekable]);
```

Comment:
`setSeekable`changes if the `throttledInterval` changes. This never happens here, but we need to add it as a dependency.